### ISSUE TITLE
Which GAN vocoders shall work with what hyper params?

### ISSUE THREAD

**Issue by skol101** [yl4579/StarGANv2-VC#35]
Am I understanding it correctly that either ParallelWaveGAN or HifiGAN shall work as vocoders? But those vocoders must be trained with the same params as the pre-trained vocoder that's provided in the demo? I followed this https://github.com/yl4579/StarGANv2-VC/issues/8#issuecomment-914651372to update preprocess.py, but not normalize.py, as, if I understand correctly, generated speakers stats have no use in StarGANv2 VC model.

I've tried first finetuning StyleMelGan from pre-trained VCTK StyleMelgan (https://github.com/kan-bayashi/ParallelWaveGAN). That didn't work out at all. 

Then I trained from scratch  StyleMelGan for about 225000 steps on the same 20 speakers that are used by StarGANv2VC. Whilst the predictions (generated wavs) by vocoder itself are excellent, when using together with the StarGAN VC model , the results weren't good. 

Both vocoder config files use the same params `n_mels=80, n_fft=2048, win_length=1200, hop_length=300,` though other hyper params are quite different from pretrained model that comes in this Demo.

**Comment by skol101** [comment]
@Kreevoz, could you share how you changed preprocess.py in the ParallelWaveGAN?

**Comment by MMMMichaelzhang** [comment]
@skol101 have you fixed out how to change the preprocess.py in the ParallelWaveGAN?

**Comment by moh3n9595** [comment]
Same issue
